tools/templates/fortran-module.txt

```diff
@@ -192,8 +192,8 @@ module {name}
 {@4}call dlite_instance_set_property_value(instance, {prop.i}, {name%c}%{prop.name})\n\
 {@elif:{prop.ndims}!0}\
 {@4}cptr = instance%get_property_by_index({prop.i})\n\
-{@4}call instance%get_property_dims_by_index({prop.i}, dims, .true.)\n\
-{@4}call c_f_pointer(cptr, {prop.name}_p, dims)\n\
+{@4}call instance%get_property_dims_by_index({prop.i}, shape, .true.)\n\
+{@4}call c_f_pointer(cptr, {prop.name}_p, shape)\n\
```

Comment:
Keep dims

Reply:
done

Reply:
wait no. This file is auto-generated with dlite-codegen. it shouldn't be changed